src/Docs/manual.tex

```diff
@@ -5000,6 +5000,7 @@ \subsection{Command}
   i,j,k by simple concatenation. (No spaces!)
 \item[-i] real from input structure file .strc instead from the output
   structure file .strc\_out.
+\item[-e] output in extended XYZ format (\url{https://wiki.fysik.dtu.dk/ase/ase/io/formatoptions.html#extxyz}).
```

Comment:
Rather than introducing the -e option, obtain the information from the -m/-c options, respectively its defaults.

Reply:
Extended XYZ is now default format for the -c option (crystal). Solved in commit [fc320c5](https://github.com/cp-paw/cp-paw/pull/8/commits/fc320c52102a0af531ac4185d64958a975ac1c75).